src/utils/responses.ts

```diff
@@ -0,0 +1,51 @@
+import * as t from "io-ts";
+import { errorsToReadableMessages } from "italia-ts-commons/lib/reporters";
+import {
+  ResponseErrorInternal,
+  ResponseErrorValidation
+} from "italia-ts-commons/lib/responses";
+
+/**
+ * Transforms async failures into internal errors
+ */
+export const withCatchAsInternalError = <T>(
+  f: () => Promise<T>,
+  message: string = "Exception while calling upstream API (likely a timeout)."
+) =>
+  f().catch(_ => {
+    // tslint:disable-next-line:no-console
+    console.error(_);
```

Comment:
is this on purpose ?

Reply:
good catch, no I think I added it before adding the logging of the responses